components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -174,6 +176,9 @@ void RRTMGPRadiation::init_buffers(const ATMBufferManager &buffer_manager)
 
 void RRTMGPRadiation::initialize_impl(const util::TimeStamp& /* t0 */) {
   using PC = scream::physics::Constants<Real>;
+  // Determine orbital year
+  m_orbital_year = m_rrtmgp_params.get<Real>("Orbital Year",1990);
```

Comment:
If `1990` is a default value, do we want to document this somewhere? Should it have a default value?

Reply:
Good question, it is the default value used by EAM.  We can definitely document this at this line.  As for whether or not we actually want a default value I would defer to @brhillman .  Presumably, if we don't have a default value we would make it so that the code throws an error if no Orbital Year is provided?

Reply:
We actually probably want to change this to allow for a dynamic orbital year. The configurations we are running right now use a perpetual orbital year, and for F-cases this looks like it’s set to 1990 in the EAM compsets for some reason. A better idea is to set the default to something that would trigger grabbing the year from the current timestamp at runtime.

Reply:
@AaronDonahue this should be fixed now, I set a default to -9999, and changed to code to grab the current year from the timestamp in the event this value is negative (the default). Can you take a look at this?